app/src/pages/inside/stepPage/modals/makeDecisionModal/copyFromHistoryLine/copyFromHistoryLine.jsx

```diff
@@ -25,12 +25,15 @@ import styles from './copyFromHistoryLine.scss';
 
 const cx = classNames.bind(styles);
 
+const HIDE_LABEL_BREAKPOINT = 850;
```

Comment:
I see the third breakpoint variable. There is a sense to put all of them in a separate file, f.e. breakpoints.js here `app/src/common/constants`.
The same questions as for breakpoints above